docs/index.md

```diff
@@ -45,205 +56,161 @@ theme:
-[![License](https://img.shields.io/pypi/l/napari.svg)](https://github.com/napari/napari/raw/master/LICENSE)
-[![Build Status](https://api.cirrus-ci.com/github/Napari/napari.svg)](https://cirrus-ci.com/napari/napari)
-[![codecov](https://codecov.io/gh/napari/napari/branch/master/graph/badge.svg)](https://codecov.io/gh/napari/napari)
-[![Python Version](https://img.shields.io/pypi/pyversions/napari.svg)](https://python.org)
-[![PyPI](https://img.shields.io/pypi/v/napari.svg)](https://pypi.org/project/napari)
-[![PyPI - Downloads](https://img.shields.io/pypi/dm/napari.svg)](https://pypistats.org/packages/napari)
-[![Development Status](https://img.shields.io/pypi/status/napari.svg)](https://github.com/napari/napari)
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/python/black)
-[![DOI](https://zenodo.org/badge/144513571.svg)](https://zenodo.org/badge/latestdoi/144513571)
+Napari is a fast multi-dimensional image viewer in Python
+and for Python. (But it *might* be useful for you outside of
+Python![*](no-python))
 
----
-
-**napari** is a fast, interactive, multi-dimensional image viewer for Python.
-It's designed for browsing, annotating, and analyzing large multi-dimensional
-images. It's built on top of `Qt` (for the GUI), `vispy` (for performant
-GPU-based rendering), and the scientific Python stack (`numpy`, `scipy`). It
-includes critical viewer features out-of-the-box, such as support for large
-multi-dimensional data, and layering and annotation. By integrating closely with
-the Python ecosystem, **napari** can be easily coupled to leading machine
-learning and image analysis tools (e.g. `scikit-image`, `scikit-learn`,
-`TensorFlow`, `PyTorch`), enabling more user-friendly automated analysis.
-
-We're developing **napari** in the open! But the project is in an **alpha**
-stage, and there will still likely be **breaking changes** with each release.
-You can follow progress
-[on our GitHub repository](https://github.com/napari/napari), test out new
-versions as we release them, and contribute ideas and code.
-
-## Installation
+```{code-cell} ipython3
+:tags: [hide-input]
+# just setting up data
+from skimage import data
+cells = data.cells3d()
+```
 
-### Which distribution to install
+```{code-cell} ipython3
+import napari
+viewer = napari.view_image(
+    cells,  # a 4D NumPy array
+    channel_axis=1,
+    ndisplay=3,
+)
+```
 
-If you want to contribute back to the napari codebase, you should install from
-source code: see the [from source](#from-source) section.
+```{code-cell} ipython3
+:tags: [remove-input]
 
-If you only wish to use napari as GUI app (and not from python code), the
-[bundled app](./tutorials/fundamentals/installation#installing-as-a-bundled-app)
-is the easiest way to install, it does not require having python pre-installed.
+from napari.utils import nbscreenshot
+viewer.camera.angles = (-30, 30, -135)
+viewer.camera.zoom = 6.5
+nbscreenshot(viewer)
```

Comment:
Do you know if we're able to add alt text to the displayed screenshot image for accessibility?

Reply:
I would love to see experiments with this too! I haven't personally run into a project that's followed through on this yet and would love to have a reference to point people towards.

Reply:
This would be nice :smiley:

Reply:
That's implemented in #3825, which just needs an update to make the input sanitising optional. We'll get there!

Reply:
That's awesome! I didn't know about that PR. Thanks for pointing it out and happy we're already thinking about it. 😄